Tests/GitHubTeams.tests.ps1

```diff
@@ -17,7 +17,490 @@ $moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
 
 try
 {
-    # TODO once more capabilities exist in the module's API set
+    # Define Script-scoped, readonly, hidden variables.
+    @{
+        defaultRepoDesc = "This is a description."
+        defaultRepoHomePage = "https://www.microsoft.com/"
+        defaultRepoTopic = "microsoft"
+    }.GetEnumerator() | ForEach-Object {
+        Set-Variable -Force -Scope Script -Option ReadOnly -Visibility Private -Name $_.Key -Value $_.Value
+    }
+
+    Describe 'GitHubTeams\Get-GitHubTeam' {
+        BeforeAll {
+            $organizationName = $script:organizationName
+        }
+
+        Context 'When getting a GitHub Team by organization' {
+            BeforeAll {
+                $teamName = [Guid]::NewGuid().Guid
+                $description = 'Team Description'
+                $privacy = 'closed'
+                $MaintainerName = $script:ownerName
+
+                $newGithubTeamParms = @{
+                    OrganizationName = $organizationName
+                    TeamName = $teamName
+                    Description = $description
+                    Privacy = $privacy
+                    MaintainerName = $MaintainerName
+                }
+
+                New-GitHubTeam @newGithubTeamParms | Out-Null
+
+                $getGitHubTeamParms = @{
+                    OrganizationName = $organizationName
+                    TeamName = $teamName
+                }
+
+                $team = Get-GitHubTeam @getGitHubTeamParms
+            }
+
+            It 'Should have the expected type and additional properties' {
+                $team.PSObject.TypeNames[0] | Should -Be 'GitHub.Team'
+                $team.name | Should -Be $teamName
+                $team.description | Should -Be $description
+                $team.organization.login | Should -Be $organizationName
+                $team.parent | Should -BeNullOrEmpty
+                $team.members_count | Should -Be 1
+                $team.repos_count | Should -Be 0
+                $team.privacy | Should -Be $privacy
+                $team.TeamName | Should -Be $teamName
+                $team.TeamId | Should -Be $team.id
+                $team.OrganizationName | Should -Be $organizationName
+            }
+
+            It 'Should support pipeline input for the organization parameter' {
+                { $team | Get-GitHubTeam -WhatIf } | Should -Not -Throw
```

Comment:
As with the other PR's, I believe we need real pipeline input tests to catch errors like there were in the past.

Reply:
Done